admin_tests/metrics/test_utils.py

```diff
@@ -52,23 +53,22 @@ class TestMetricsGetDaysStatistics(AdminTestCase):
     def setUp(self):
         super(TestMetricsGetDaysStatistics, self).setUp()
         Node.remove()
-        NodeFactory(category='project')  # makes 2 nodes
-        NodeFactory(category='data')  # probably makes 1 data 1 project?
-        print get_projects()
+        NodeFactory(category='project')  # makes Node, plus parent
```

Comment:
Not gonna lie: I kind of hate the NodeFactory and the need to have a comment about what it's doing. How about we replace it with something more explicit, like a project factory and then add a component to it? Since we're messing around here anyways.